test-scripts/integrationTest.ts

```diff
@@ -3,11 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { join } from 'path'
+import * as path from 'path'
```

Comment:
nit: why not just import resolve instead?

Reply:
It's my default coding style... I've changed it.